scripts/python/examples/read_ascii_point.py

```diff
@@ -0,0 +1,62 @@
+import os
+import sys
+
+from met.point import met_point_tools as tools
+
+########################################################################
+
+print("Python Script:\t" + repr(sys.argv[0]))
+
+##
+##  input file specified on the command line
+##  load the data into the numpy array
+##
+
+arg_cnt = len(sys.argv)
+if arg_cnt < 2:
+   print("ERROR: read_ascii_point.py -> Missing an input file.")
+   sys.exit(1)
+
+do_convert = False
+last_index = 2
+if arg_cnt > last_index:
+   opt_convert = sys.argv[2]
+   if opt_convert.lower() == "do_convert" or opt_convert.lower() == "convert":
+      do_convert = True
+      last_index += 1
```

Comment:
Since changes in this PR allow MET to support both `point_data` and `met_point_data`, do we need this code anymore?

Reply:
No for the operation. It's only for internal/manual test for conversion.